docs/elements.md

```diff
@@ -1256,6 +1257,60 @@ See the [`pl-drawing` documentation](pl-drawing/index.md) for details.
 
 -----
 
+## `pl-overlay` element
+
+The overlay element allows existing PrairieLearn and HTML elements to be layered on top of one another in arbitrary positions.
```

Comment:
Creative re-use of the `pl-drawing` capabilities. 

Maybe emphasize this doesn't allow for an interactive drag/drop approach?